taichi/ui/backends/vulkan/app_context.h

```diff
@@ -6,16 +6,23 @@
 #include "taichi/backends/vulkan/vulkan_device.h"
 #include "taichi/ui/backends/vulkan/swap_chain.h"
 
+#ifdef ANDROID
+#include <android/native_window.h>
+typedef ANativeWindow TaichiWindow;
+#else
+typedef GLFWwindow TaichiWindow;
+#endif
```

Comment:
tiny nit, let's use `using A = B` instead of `typedef B A`, and maybe put `TaichiWindow` inside `taichi::ui`?

Reply:
Done! My habits of C developer :), I left the `#include` outside as well to avoid having all the functions from this header inside the Taichi namespace, I think I could have used `namespace { #include <android/native_window.h> }` instead to avoid the extra `ifdef ANDROID` but not sure what is preferred here.

Reply:
I think this is fine.